demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -551,7 +552,14 @@ def start_mdx_server(handle_error: Optional[Callable] = None, file_path: Optiona
 
                     else:
                         raise Exception(error_message)
-        return True
+        yield True
+        ReadMeValidator.stop_mdx_server()
+
+    @staticmethod
+    def add_node_env_vars():
+        content_path = get_content_path()
+        node_modules_path = content_path / Path('node_modules')
```

Comment:
```suggestion
        node_modules_path = content_path / 'node_modules'
```
if content_path is a `Path` object, no need to call `Path` on the other, `/` will work on strings too

Reply:
content_path is a str

Reply:
cool, i didn't know you could do `str / Path` -> `Path`